librz/core/cdebug.c

```diff
@@ -587,18 +587,27 @@ RZ_API void rz_core_debug_ri(RzCore *core, RzReg *reg, int mode) {
 
 RZ_IPI void rz_core_debug_sync_bits(RzCore *core) {
 	if (rz_config_get_b(core->config, "cfg.debug")) {
+		ut64 asm_bits = rz_config_get_i(core->config, "asm.bits");
 		switch (core->dbg->bits) {
```

Comment:
You can simplify into something like `if (core->dbg->bits == asm_bits) { return; } switch ...`

Reply:
i agree

Reply:
Well the units are slightly different, but ok 86e35c4acf3f65da1269d42a8d3324ffaf26e4af